src/const.ts

```diff
@@ -160,31 +158,6 @@ export const ORDER_BOOK_HOPS_MAX = 30
 // https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1474.md
 export const LIMIT_EXCEEDED_ERROR_CODE = -32005
 
-function getSubgraphUrls(): Partial<Record<ChainId, string>> {
-  const subgraphBaseUrls: Partial<Record<ChainId, string>> = {}
-  const [mainnetUrl, gcUrl, goerliUrl] = [
-    process.env.REACT_APP_SUBGRAPH_URL_MAINNET || undefined,
-    process.env.REACT_APP_SUBGRAPH_URL_GNOSIS_CHAIN || undefined,
-    process.env.REACT_APP_SUBGRAPH_URL_GOERLI || undefined,
-  ]
-
-  if (mainnetUrl) {
-    subgraphBaseUrls[ChainId.MAINNET] = mainnetUrl
-  }
-  if (gcUrl) {
-    subgraphBaseUrls[ChainId.GNOSIS_CHAIN] = gcUrl
-  }
-  if (goerliUrl) {
-    subgraphBaseUrls[ChainId.GOERLI] = goerliUrl
-  }
-
-  return subgraphBaseUrls
-}
-
-export const COW_SDK = new CowSdk(Network.MAINNET, {
-  subgraphBaseUrls: getSubgraphUrls(),
-})
```

Comment:
Is this logic moving somewhere else?
Otherwise, how will the subgraph urls be passed onto the subgraph instance?

Reply:
Thank you!
Added it:
https://github.com/cowprotocol/explorer/pull/391/commits/005c1c39d5cd6d1643abbe2233434a6d80a72887